api/lib/domain/constants.js

```diff
@@ -46,6 +46,8 @@ module.exports = {
         email: { shortCode: 'R31', code: 'ACCOUNT_WITH_EMAIL_ALREADY_EXIST_FOR_THE_SAME_ORGANIZATION' },
         username: { shortCode: 'R32', code: 'ACCOUNT_WITH_USERNAME_ALREADY_EXIST_FOR_THE_SAME_ORGANIZATION' },
         samlId: { shortCode: 'R33', code: 'ACCOUNT_WITH_GAR_ALREADY_EXIST_FOR_THE_SAME_ORGANIZATION' },
+        anotherStudentIsAlreadyReconciled: { shortCode: 'R70', code: 'ACCOUNT_WITH_ANOTHER_STUDENT_EXIST_FOR_THE_SAME_ORGANIZATION' },
```

Comment:
Prof d'anglais "An account with another student already exists for the same organization"
```suggestion
        anotherStudentIsAlreadyReconciled: { shortCode: 'R70', code: 'ACCOUNT_WITH_ANOTHER_STUDENT_EXISTS_FOR_THE_SAME_ORGANIZATION' },
```
Je dirais même 
`USER_ALREADY_RECONCILED_IN_THIS_ORGANIZATION`